libs/types/models/ConditionType.ts

```diff
@@ -4,17 +4,14 @@
 /* eslint-disable */
 export enum ConditionType {
   EnrollmentRequestApproved = 'Approved',
-  CertificateSigningRequestApproved = 'Approved',
-  CertificateSigningRequestDenied = 'Denied',
-  CertificateSigningRequestFailed = 'Failed',
-  RepositoryAccessible = 'Accessible',
-  ResourceSyncAccessible = 'Accessible',
-  ResourceSyncResourceParsed = 'ResourceParsed',
-  ResourceSyncSynced = 'Synced',
-  FleetOverlappingSelectors = 'OverlappingSelectors',
-  FleetValid = 'Valid',
-  DeviceUpdating = 'Updating',
-  DeviceSpecValid = 'SpecValid',
-  DeviceMultipleOwners = 'MultipleOwners',
-  TemplateVersionValid = 'Valid'
+  CertificateSigningRequestApproved = 'Denied',
+  CertificateSigningRequestDenied = 'Failed',
+  CertificateSigningRequestFailed = 'Accessible',
+  RepositoryAccessible = 'ResourceParsed',
+  ResourceSyncAccessible = 'Synced',
+  ResourceSyncResourceParsed = 'OverlappingSelectors',
+  ResourceSyncSynced = 'Valid',
+  FleetOverlappingSelectors = 'Updating',
+  FleetValid = 'SpecValid',
+  DeviceUpdating = 'MultipleOwners',
```

Comment:
These seem broken completely, WDYT @celdrake  ?

Reply:
Yes, since the API defines duplicate keys, the enum shifts.

The only solution I found is to keep reverting the file, and only update it when there's a change in the API definitions (requires making manual changes).

I checked the generator back in the day and didn't found a way to fix the problem from root.